src/wfs/src/main/java/org/geoserver/wfs/json/GeoJSONGetFeatureResponse.java

```diff
@@ -460,6 +461,11 @@ private FeaturesInfo encodeSimpleFeatures(
                                     jsonWriter.writeGeom((Geometry) value);
                                 }
                             }
+                        } else if (Date.class.isAssignableFrom(ad.getType().getBinding())
```

Comment:
This is for WFS GetFeature, not for WMS GetFeatureInfo. Isn't this out of scope?

Reply:
In fact this is a bit confusing at first sight, but doing some debugging this WFS class is used by WMS GetFeatureInfo JSON format output as we can see on stack trace obtained on ```org.geoserver.wms.featureinfo.GetFeatureInfoJSONTest.testDateTimeFormattingEnabled()``` test execution:

```
GeoJSONGetFeatureResponse.encodeSimpleFeatures(GeoJSONBuilder, List<FeatureCollection>, boolean, Operation) line: 467	
GeoJSONGetFeatureResponse.writeFeatures(List<FeatureCollection>, Operation, boolean, GeoJSONBuilder) line: 210	
GeoJSONGetFeatureResponse.write(FeatureCollectionResponse, OutputStream, Operation) line: 133	
GeoJSONGetFeatureResponse(WFSGetFeatureOutputFormat).write(Object, OutputStream, Operation) line: 200	
GeoJSONFeatureInfoResponse.write(FeatureCollectionType, GetFeatureInfoRequest, OutputStream) line: 49	
GetFeatureInfoResponse.write(Object, OutputStream, Operation) line: 99	
Dispatcher.response(Object, Request, Operation) line: 1031	
```

Reply:
Fine, I was suspecting that, but needed a confirm :-)